---
title: SanitizationException.SanitizationException
second_title: Aspose.PDF for .NET API Reference
description: Constructeur de SanitizationException. Initialise une nouvelle instance de la classe SanitizationException
type: docs
weight: 10
url: /fr/net/aspose.pdf.sanitization/sanitizationexception/sanitizationexception/
---
## SanitizationException() {#constructor}

Initialise une nouvelle instance de la classe [`SanitizationException`](../).

```csharp
public SanitizationException()
```

### Voir aussi

* classe [SanitizationException](../)
* espace de noms [Aspose.Pdf.Sanitization](../../../aspose.pdf.sanitization/)
* assembly [Aspose.PDF](../../../)

---

## SanitizationException(string) {#constructor_2}

Initialise une nouvelle instance de la classe [`SanitizationException`](../).

```csharp
public SanitizationException(string message)
```

| Paramètre | Type | Description |
| --- | --- | --- |
| message | String | Le message. |

### Voir aussi

* classe [SanitizationException](../)
* espace de noms [Aspose.Pdf.Sanitization](../../../aspose.pdf.sanitization/)
* assembly [Aspose.PDF](../../../)

---

## SanitizationException(string, Exception) {#constructor_3}

Initialise une nouvelle instance de la classe [`SanitizationException`](../) avec un message d'erreur spécifié et une référence à l'exception interne qui est la cause de cette exception.

```csharp
public SanitizationException(string message, Exception innerException)
```

| Paramètre | Type | Description |
| --- | --- | --- |
| message | String | Le message d'erreur qui explique la raison de l'exception. |
| innerException | Exception | L'exception qui est la cause de l'exception actuelle, ou une référence nulle (Nothing en Visual Basic) si aucune exception interne n'est spécifiée. |

### Voir aussi

* classe [SanitizationException](../)
* espace de noms [Aspose.Pdf.Sanitization](../../../aspose.pdf.sanitization/)
* assembly [Aspose.PDF](../../../)

---

## SanitizationException(Exception) {#constructor_1}

Initialise une nouvelle instance de la classe [`SanitizationException`](../) avec un message d'erreur spécifié et une référence à l'exception interne qui est la cause de cette exception.

```csharp
public SanitizationException(Exception innerException)
```

| Paramètre | Type | Description |
| --- | --- | --- |
| innerException | Exception | L'exception qui est la cause de l'exception actuelle, ou une référence nulle (Nothing en Visual Basic) si aucune exception interne n'est spécifiée. |

### Voir aussi

* classe [SanitizationException](../)
* espace de noms [Aspose.Pdf.Sanitization](../../../aspose.pdf.sanitization/)
* assembly [Aspose.PDF](../../../)